### DISCUSSION TITLE
question about other 95 pet

### DISCUSSION THREAD

**Discussion by zmb9332** [Adrianotiger/desktopPet#122] [Ideas]
Back in the 90's, there were two pets, a sheep and an orange, made by the same company. If I provided the original 16 bit orange program do you think you'd be able to implement it?

https://drive.google.com/file/d/1wO0E1Tbwjp3S8OM34ECAI4YjMzi4Ye8g/view?usp=sharing

you need a virtual machine with at least windows xp to view it, on a separate note: some of the behaviors of sheep seem to not be present like eating grass, spawning the black sheep that play together, in case you didnt have the original sheep file to observe, i have also included a link to it

https://drive.google.com/file/d/1pcuV8mSLWDssT4Tw90HWSRMQ1bA85hLq/view?usp=sharing

**Comment by Adrianotiger** [comment]
Hi.
There was someone who was able to retrieve all images.
But I never got any reply...

See: https://github.com/Adrianotiger/desktopPet/issues/33
